src/kafka_consumer.cpp

```diff
@@ -174,26 +185,26 @@ RdKafka::ErrorCode msg_consume(RdKafka::Message* message, void* opaque, BSMHandl
             msg_cnt++;
             msg_bytes += message->len();
             if (verbosity >= 3) {
-                std::cerr << "Read msg at offset " << message->offset() << "\n";
+                cerr << "Read msg at offset " << message->offset() << "\n";
```

Comment:
How much effort to swap this class over to using a logger rather than cout/cerr?

Reply:
There is an internal and an external logger specified in the PPM class, so they can only be used in methods within that class. If we wanted to use the logger here we'd probably have to do some refactoring and create a shared logger class that can be utilized by the entire project.

Reply:
Should I create a work item for this?

Reply:
This can tie into the other comment with adding to the backlog.